docs/how-to-setup-freecodecamp-mobile-app-locally.md

```diff
@@ -367,6 +367,114 @@ Follow these steps:
     git push origin branch/name-here
     ```
 
+## Running mobile curriculum tests
+
+> [!NOTE]
+> You need to follow this section if you're modifying the challenge test runner in the mobile app otherwise you can go ahead to the next section on how to open a Pull Request.
```

Comment:
```suggestion
> You only need to follow this section if you're modifying the challenge test runner in the mobile app. Otherwise, you can go to the next section on [how to open a pull request](#proposing-a-pull-request-pr).
```

Grammar, plus we can hot-link.